editor/Makefile

```diff
@@ -67,6 +74,12 @@ gen/js/proto/ord/%.js : ../ord_schema/proto/%.proto
 ../build/lib/ord_schema/proto/%_pb2.py : ../ord_schema/proto/%.proto
 	$(error $< not found--run setup?)
 
+# Checks for the presense of the last file produced in Ketcher's build. 
+# (hacky; may want to revisit later)
+ketcher/dist/doc/help.html : 
+	# Ensure that ketcher has been built.
+	$(error built ketcher not found, see editor readme?)
```

Comment:
I think you can omit this target. The tag check is sufficient.

Reply:
I was worried about the Ketcher build failing, but users still trying to proceed anyways. Is this worth checking for?

Reply:
But the ketcher build is not part of this build.

I wouldn't use make to ensure that a human did a job.

Reply:
Makes sense, sounds good.

Thanks for the checks!